README.md

```diff
@@ -2,12 +2,23 @@
 
 An interactive tour of the Gleam programming language.
 
-```sh
-# Download a wasm version of the Gleam compiler
-./bin/download-compiler
+## Setup
+
+In order to run this project you will need the following prerequisites: 
+
+ - Node.js (https://nodejs.org/en/download)
+   - http-server (https://www.npmjs.com/package/http-server)
+ - Gleam (https://gleam.run/getting-started/installing/)
+ - Gleam's wasm compiler (use `./bin/download-compiler`)
 
-# Build the site
+_Node: Codespace users will have this installed automatically_
+
+## Running the site
+
+```sh
+# Running the project will generate a static site under the `public/` folder
 gleam run
 
-# It's now all the in `public/` directory
-```
+# Running the server will serve the newly created static files
+http-server public/
+```
```

Comment:
Could you revert the changes to this file please? Thank you.

Reply:
Done :)